### ISSUE TITLE
Svg icon export error

### ISSUE THREAD

**Issue by Vargentum** [avocode/avocode#351]
Error occurs when trying to export rasterised icon from synced Sketch file.

```
TypeError: boolean is not a function
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/export-panel/lib/stores/export.js:1:11098
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/export.js:1:4967
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/fstream/node_modules/graceful-fs/graceful-fs.js:104:5
    at FSReqWrap.oncomplete (fs.js:77:15)
```

**Comment by rychlis** [comment]
Hello, sorry for not responding earlier. Could you try if you can replicate the issue on Avocode 2.3.2?

**Comment by rychlis** [comment]
looks like someone had the same issue in 2.3.2 - #366

**Comment by rychlis** [comment]
Also in 2.4.1 #385

**Comment by petrbrzek** [comment]
Hello, we’ve just released Avocode 2.4.3 that should fix the issue. Let us know if the problem persists.
